README.md

```
@@ -110,6 +110,54 @@ server {
+TEST_DB_HOST - Test database host
+TEST_DB_PORT - Test database port
+PROD_DB_USERNAME - Production database username
+PROD_DB_PASSWORD - Production database password
+PROD_DB_NAME - Production database name
+PROD_DB_HOST - Production database host
+PROD_DB_PORT - Production database port
+```
+
+2. Email service configuration
+For the email service to run correctly, the user should add their own email credentials
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)